src/pages/index.js

```diff
@@ -31,15 +31,16 @@ const P = styled.p`
       : ""}
 `;
 
-const Index = () => {
+const Index = ({ userCount }) => {
   return (
     <Layout title="Welcome" homepage>
       <h1>
         <span>Welcome to</span>
         Reactiflux
       </h1>
       <p>
-        We’re a chat community of 170,000+ React&nbsp;JS&nbsp;
+        We’re a chat community of{" "}
+        {new Intl.NumberFormat("en-US").format(userCount)} React&nbsp;JS&nbsp;
```

Comment:
Are we intentionally mixing non-breaking spaces and JSX expressions here? I know I'm a massive pedant but would it be more consistent to use a non-breaking space instead of the JSX expression here?

Reply:
Also, is there a reason you prefer `Intl.NumberFormat` over `Number.toLocaleString`? I'm genuinely curious, not suggesting a change.

Reply:
The mix of whitespace syntax is part prettier and part old copy.

No preference; Intl is the first thing that came to mind to format the number

Reply:
Fair enough, looks good to me either way 👍

Reply:
I believe the non-breaking spaces were intentionally placed here to force "React JS ⚛️" to appear on one line, without forcing breaks around the user count.

Reply:
Ah, that'd make sense!